docs/source/AdministratorGuide/Systems/DataManagement/index.rst

```diff
@@ -160,7 +160,30 @@ GFAL2_SRM2 plugins because only srm is allowed as a write plugin, but since this
 
 
 The WriteProtocols and AccessProtocols list can be locally overwritten in the SE definition.
-  
+
+
+
+--------------------------
+Available protocol plugins
+--------------------------
+
+DIRAC comes with a bunch of plugins that you can use to interact with StorageElements.
+These are the plugins that you should define in the `PluginName` option of your StorageElement definition.
+
+  - DIP: used for dips, the DIRAC custom protocol (useful for example for DIRAC SEs).
+  - File: offers an abstraction of the local access as an SE.
+  - SRM2: for the srm protocol, using the deprecated gfal libraries.
+  - RFIO: for the rfio protocol.
+  - Proxy: to be used with the StorageElementProxy.
+  - XROOT: for the xroot protocol, using the python xroot binding.
+
+There are also a set of plugins based on the gfal2 libraries.
```

Comment:
Maybe just say which version introduced them. It would also not be bad to add links to "XROOT" or "GFAL2" documentations, if they exist.